ui/app/styles/components/allocation-fs.scss

```diff
@@ -0,0 +1,70 @@
+.fs-explorer.table {
+  width: 100%;
+
+  tbody {
+    a {
+      text-decoration: none;
+      color: inherit;
+
+      &:hover {
+        .name {
+          text-decoration: underline;
+        }
+      }
+    }
+  }
+}
+
+.breadcrumb.fs {
+  margin: 0;
+
+  a {
+    padding-top: 0;
+    padding-bottom: 0;
+    color: $blue;
+    opacity: 1;
+    font-weight: $weight-bold;
+    text-decoration: none;
+
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+
+  .is-active a {
+    color: $black;
+  }
+}
```

Comment:
One aspect of the breadcrumbs I caught was that the `/` divider is still green like the primary breadcrumbs. Can you make it one of the many grays we have instead? I don't have an exact gray to suggest, just something that contrasts enough to standout even if you have low vision but is simultaneously less prominent than the breadcrumbs themselves.

Reply:
hah wow, I completely missed that. I changed it to `$grey-light`:

![image](https://user-images.githubusercontent.com/43280/60526612-4c530180-9cb6-11e9-9ca3-e9be8258aa24.png)